python/zfpy.pyx

```diff
@@ -245,25 +246,24 @@ cdef _validate_4d_list(in_list, list_name):
         )
 
 cpdef np.ndarray _decompress(
-    bytes compressed_data,
+    const uint8_t[::1] compressed_data,
     zfp_type ztype,
     shape,
     out=None,
     double tolerance = -1,
     double rate = -1,
     int precision = -1,
 ):
-
     if compressed_data is None:
         raise TypeError("compressed_data cannot be None")
     if compressed_data is out:
         raise ValueError("Cannot decompress in-place")
     _validate_4d_list(shape, "shape")
 
-    cdef char* comp_data_pointer = compressed_data
+    cdef const char* comp_data_pointer = <const char*>&compressed_data[0]
```

Comment:
Can we please change this to `const void*` to avoid any confusion why the pointer is initially cast to a (signed) `char*`?